### PR TITLE
Intro champions 2024

### PR DESCRIPTION
**Pull request by yabellini**

### CODE REVIEW

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+- Mirna Vazquez Rosas Landa
+- Erika Siregar
+- Jacqui Levy
+- Yanina Bellini Saibene
+date: '2024-02-13'
+slug: champions-program-champions-2024
+tags:
+- community
+- champions program
+description:
```
Comment (steffilazerte) [inline content/blog/2024-02-13-intro-champions-2024/index.md:20]: description?

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+- Andrea Gomez Vargas
+- Liz Hare
+- Francesca Belem Lopes Palmeira
+- Binod Jung Bogati
+- Sunny Tseng
+- Mirna Vazquez Rosas Landa
+- Erika Siregar
+- Jacqui Levy
+- Yanina Bellini Saibene
+date: '2024-02-13'
```
Comment (steffilazerte) [inline content/blog/2024-02-13-intro-champions-2024/index.md:15]: just a reminder to update here and the folder name 😁

**Review by steffilazerte — COMMENTED** [review]
Wow, what an amazing cohort!
Looks good, just a couple of suggestions and some wording tweaks. 
I can't wait to see what this year's group gets up to!

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [ ] publication date is ok -> reminder to update
* [X] alternative text of images is informative
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)
* [X] YAML field 'preface' is present if necessary

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
+---
+title: Introducing rOpenSci Champions - Cohort 2023-2024
+author:
+- Ezekiel Adebayo Ogundepo
+- Sehrish Kanwal
+- Andrea Gomez Vargas
+- Liz Hare
+- Francesca Belem Lopes Palmeira
+- Binod Jung Bogati
+- Sunny Tseng
```
Comment (yabellini) [inline content/blog/2024-02-13-intro-champions-2024/index.md:10]:
```suggestion
- Yi-Chin Sunny Tseng"
```

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+- Mirna Vazquez Rosas Landa
+- Erika Siregar
+- Jacqui Levy
+- Yanina Bellini Saibene
+date: '2024-02-13'
+slug: champions-program-champions-2024
+tags:
+- community
+- champions program
+description:
```
Comment (yabellini) [inline content/blog/2024-02-13-intro-champions-2024/index.md:20]:
```suggestion
description: 
The rOpenSci Champions Program starts this 2024 with a new cohort of Champions.  We are pleased to introduce you to our Champions and their projects!
```

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+
+Stay tuned for more updates on their progress!
+
+
+I am Erika Siregar, a data science enthusiast from Jakarta, Indonesia. With an academic background in Statistics and Computer Science, I actively contribute to data-driven insights at BPS-Statistics Indonesia and consult on R Shiny for the bioenergy project with GIZ Indonesia. As the co-founder of R-Ladies Jakarta, I've observed a strong interest in R among women and gender minorities, highlighting the crucial need for accessible resources. In the rOpenSci Champions Program, my project involves developing "rplaywright," an R package seamlessly bridging to Playwright, simplifying web testing and automation for R users. I eagerly anticipate gaining new skills, building connections, and making a meaningful impact in the vibrant rOpenSci community. With my PhD journey in Information Science at the University of Sheffield beginning this April, I envision utilizing my learning and experience in the rOpenSci Champions Program to fortify and advocate for open-source collaboration in my research pursuits.
+
+## Jacqui Levy
```
[the remark this answers is not in the document]
Comment (yabellini) [reply content/blog/2024-02-13-intro-champions-2024/index.md:112]: Yes, I agree.  I can use the information in her application.

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+- Andrea Gomez Vargas
+- Liz Hare
+- Francesca Belem Lopes Palmeira
+- Binod Jung Bogati
+- Yi-Chin Sunny Tseng"
+- Mirna Vazquez Rosas Landa
+- Erika Siregar
+- Jacqui Levy
+- Yanina Bellini Saibene
+date: '2024-02-13'
```
Comment (yabellini) [inline content/blog/2024-02-13-intro-champions-2024/index.md:15]:
```suggestion
date: '2024-02-15'
```

File: content/blog/2024-02-13-intro-champions-2024/index.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+## Jacqui Levy
+
+{{< figure src = "/img/team/jacqui-levy.png" width = "300" alt = "Profile photo of Jacqui Levy." caption = "<strong>Jacqui Levy - Environment and Climate Change Canada, rOpenSci</strong>">}}
+
+I am a data analyst with a background in environmental science, and I am a data enthusiast interested in all things to do with data and open source science. I am looking forward to learning about software development and being part of the welcoming community of Champions and mentors with rOpenSci.
```
Comment (yabellini) [inline content/blog/2024-02-13-intro-champions-2024/index.md]:
```suggestion
I am a data analyst with a background in environmental science and a data enthusiast interested in all things to do with data and open-source science. I have worked developing tools to standardize datasets and reporting information related to climate change using R. I am looking forward to learning about software development and being part of the welcoming community of Champions and mentors with rOpenSci. 
```

**Comment by yabellini** [comment]
Thank you @steffilazerte ! All changes done. :-)

**Pull request merged**